### DISCUSSION TITLE
Storybook with Vue Custom Elements

### DISCUSSION THREAD

**Discussion by oliveirarafa** [storybookjs/storybook#22292] [Help]
Hi. I'm using storybook to show my Vue componentes library. I'm creating using custom elements (componente.ce.vue) but i cant make my apps visible. For what I figured my styles don't get injected inside the shadow dom.

The HTML for my component is on the page but the css is lacking.

How might i debug this? I cant figure it out.

**Comment by f3bw** [comment]
Have you added the style import statement into the .mdx file or the main.js storybook file?

**Reply by oliveirarafa** [reply]
Yeah, I don't have access to the code right now but if I remember correctly I managed to make it work by adding my library to the manager.js.
